aioshelly/block_device.py

```diff
@@ -268,6 +268,17 @@ async def trigger_shelly_gas_unmute(self) -> None:
         """Trigger a Shelly Gas unmute action."""
         await self.http_request("get", "unmute")
 
+    async def enable_shelly_motion_detection(self, enable: bool) -> None:
+        """Enable or disable Shelly Motion motion detection."""
+        params = {}
+
+        if enable:
+            params = {"motion_enable": "true"}
+        else:
+            params = {"motion_enable": "false"}
+
```

Comment:
```suggestion
        params = {"motion_enable": "true"} if enable else {"motion_enable": "false"}
```

Reply:
I was not sure if you write it in long or short form. Thank you for suggestion, I changed it.